package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "debugger-for-edge",
```

Comment:
Any idea why is this red?

Reply:
...no idea

Reply:
https://github.com/sindresorhus/refined-github/issues/1251  Looks like GitHub telling us it sees a Syntax error..

Reply:
So, from what the code-eol extension is showing, my package.json line endings were changed to lf in my last prev. commit (this is from checking out my prev. commit)
![oldlineendings](https://user-images.githubusercontent.com/16889261/46105333-a7d1eb00-c18a-11e8-94b9-63d57978c902.png)

And with the push I just made, they are now crlf (locally, on my current commit)
![newlineendings](https://user-images.githubusercontent.com/16889261/46105475-ee274a00-c18a-11e8-996a-926eae74c7fd.png)

Reply:
And from what I am understanding, everything should be in crlf, because the rest of the code has crlf for line endings
![example1](https://user-images.githubusercontent.com/16889261/46105623-6130c080-c18b-11e8-9ff0-7a01c90d29c3.png)
![example2](https://user-images.githubusercontent.com/16889261/46105626-62fa8400-c18b-11e8-9edd-4b2c226c5fa0.png)